scripts/completion/dvc.bash

```diff
@@ -82,13 +86,21 @@ _dvc () {
       *)  COMPREPLY=($(compgen -W "$_dvc_commands" -- "$word")) ;;
     esac
   elif [ "${COMP_CWORD}" -eq 2 ]; then
-    local options_list="_dvc_${COMP_WORDS[1]}"
+    if [[ ${COMP_WORDS[1]:0:1} == "-" ]]; then
+      return 0
+    else
+      local options_list="_dvc_$(replace_hyphen ${COMP_WORDS[1]})"
 
-    COMPREPLY=($(compgen -W "$_dvc_global_options ${!options_list}" -- "$word"))
+      COMPREPLY=($(compgen -W "$_dvc_global_options ${!options_list}" -- "$word"))
+    fi
   elif [ "${COMP_CWORD}" -eq 3 ]; then
-    local options_list="_dvc_${COMP_WORDS[1]}_${COMP_WORDS[2]}"
+    if [[ ${COMP_WORDS[2]:0:1} == "-" ]]; then
+      return 0
+    else
+      local options_list="_dvc_${COMP_WORDS[1]}_$(replace_hyphen ${COMP_WORDS[2]})"
```

Comment:
You need to replace hyphen in `COMP_WORDS[1]` here too, otherwise you'll get an error on `import-url` and `get-url`, because it would evaluate to `_dvc_import-url_...`, which is invalid.